Mollier/SAM.Core.Mollier.UI/Modify/CreateMovedLabel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using SAM.Core.Mollier.UI.Controls;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.DataVisualization.Charting;
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Modify
+    {
+        [Obsolete("To be Changed Maciek")] // TODO: Change (MollierProcess)
+        public static void CreateMovedLabel(Chart chart, MollierControlSettings mollierControlSettings, double X, double Y, int Mollier_angle, int Psychrometric_angle, double Mollier_X, double Mollier_Y, double Psychrometric_X, double Psychrometric_Y, string LabelContent, ChartDataType chartDataType, ChartParameterType chartParameterType, bool fontChange = false, Color? color = null, string tag = null)
```

Comment:
Probably move to Create?? 
return List<Series>

To think about name: Example Series_Text